mindjunkies/forums/tests/test_models.py

```
@@ -3,125 +1,204 @@
+            module=self.module
         )
-        last_reply = (
-            Reply.objects.filter(parent_comment__topic=self.topic)
-            .order_by("-created")
-            .first()
+        topics = ForumTopic.objects.all()
+        self.assertEqual(topics[0], topic2)  # Newer topic first
+
+    def test_forum_comment_ordering(self):
+        """Test ForumComment Meta ordering"""
+        comment2 = ForumComment.objects.create(
```

Ruff (0.8.2)

Local variable `comment2` is assigned to but never used

Remove assignment to unused variable `comment2`

(F841)